Cocoaheads School_Lesson2.playground/Contents.swift

```diff
@@ -36,7 +63,7 @@ class StackStatistics: Stack {
         }
     }
     
-    override func pop() -> Int? {
+    override func pop() -> Element? {
```

Comment:
```suggestion
    override func pop() -> Element? {
```
I think it is necessary to add a description for this method

Reply:
done